CONTRIBUTING.md

```diff
@@ -0,0 +1,10 @@
+## General contribution guidelines
+
+* Have a look at open issues. Pick an unassigned issue. If doubtful, discuss on the issue itself.
+* You can also create a new issue for a feature or bug.
+* Make sure you are assigned to the issue before you start working on it.
+* Make sure you are not working on more than 2 issues at once.
+* While sending a PR make sure you follow one issue per PR rule.
+* Squash your commits in all PRs you send.
```

Comment:
I think commits should be squashed once PR is approved, at the time of merging. That can be done by any of the collaborators. Squash is necessary when there are too many commits. So please consider removing this point.